tests/test_cms_plugins.py

```diff
@@ -0,0 +1,76 @@
+from cms.api import add_plugin, create_page
+from cms.test_utils.testcases import CMSTestCase
+
+from aldryn_forms.models import FormSubmission
+
+
+class FormPluginTestCase(CMSTestCase):
+    def setUp(self):
+        super(FormPluginTestCase, self).setUp()
+
+        self.page = create_page('test page', 'INHERIT', 'en', published=True)
```

Comment:
this is a bit dangerous.
I suggest to create a template for this and use it explicitly